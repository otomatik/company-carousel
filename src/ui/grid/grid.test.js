import { Grid } from './';
import { shallow } from 'enzyme';
import React from 'react';

describe('Grid', () => {
  it('should match snapshot', () => {
    // when
    const grid = shallow(<Grid columns={2} rows={2} currentArea={0} />);

    // then
    expect(grid).toMatchSnapshot();
  });
});
