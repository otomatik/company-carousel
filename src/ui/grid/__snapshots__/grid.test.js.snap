// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should match snapshot 1`] = `
<styled.div
  columns={2}
  currentArea={0}
  rows={2}
  totalColumns={6}
>
  <Tile
    color="orange"
  />
  <Tile
    color="red"
  />
  <Tile
    color="green"
  />
  <Tile
    color="blue"
  />
  <Tile
    color="violet"
  />
  <Tile
    color="black"
  />
  <Tile
    color="gray"
  />
  <Tile
    color="purple"
  />
  <Tile
    color="yellow"
  />
</styled.div>
`;
